src/variety/tentaisho.js

```diff
@@ -254,6 +254,7 @@
 
 			var cell = this.validcell();
 			if (cell) {
+				this.puzzle.board.roommgr.rebuild();
```

Comment:
How is this related to the series of answer inputs that the bug report speaks of? Dot is never changed when you're playing.